packages/webpack-extraction-plugin/src/sortCSSRules.test.ts

```diff
@@ -135,4 +135,17 @@ describe('sortCSSRules', () => {
       }
     `);
   });
+
+  it('removes comments from output', () => {
+    const css = `
+      /* This is a comment in CSS */
```

Comment:
> When Webpack emits assets it can add a comment to source if [output.pathinfo](https://webpack.js.org/configuration/output/#outputpathinfo) is enabled.

Check PR's description :)

Reply:
I mean what comments are they ? are they comments from JS code that are in the `makeStyles` call ?

Reply:
ah ok would be nice to add that to the issue description.

Reply:
Updated the issue with more details 👍